docs/src/components/pricing/PricingTable.tsx

```diff
@@ -19,17 +19,19 @@ const planInfo = {
     color: 'green',
     title: 'Community',
     description:
-      'Get started with the industry-standard UI library for building React user interfaces (open-source, MIT licensed).',
+      'Get all the MUI component libraries, MIT-licensed, open source/open core, and free forever.',
```

Comment:
@samuelsycamore double-checking on the grammar here, is "Get all the MUI component libraries" correct?

Reply:
I would drop the imperative and just go with a simple statement:

```suggestion
      'All of MUI's component libraries are MIT-licensed, open source/open core, and free forever.',
```

Reply:
I like the original mention to the industry-standard library.
WDYT about:

```suggestion
      'All the MIT-licensed MUI components. The industry-standard UI library for building React user interfaces',
```